source/speech/speech.py

```diff
@@ -338,6 +339,40 @@ def _getSpellingSpeechWithoutCharMode(
 		yield EndUtteranceCommand()
 
 
+def getSingleCharDescription(
+		text: str,
+		locale: Optional[str] = None,
+) -> Generator[SequenceItemT, None, None]:
+	# This should only be used for single chars.
+	if not len(text) == 1:

```

Comment:
Does len(foo)==1 work for langs with chars that are not ascii in all cases?

Reply:
Yes

>The length of a single Unicode character as a Python str will always be 1, no matter how many bytes it occupies.

[Source with example you can test](https://realpython.com/python-encodings-guide/#one-byte-two-bytes-three-bytes-four)